pkg/scannerv4/mappers/mappers.go

```diff
@@ -405,6 +456,17 @@ func toProtoV4VulnerabilitiesMap(ctx context.Context, vulns map[string]*claircor
 			// The preferred CVSS metrics will always be stored at the first index.
 			preferredCVSS = metrics[0]
 		}
+
+		var vulnEPSS *epss.EPSSItem
+		if epss, ok := epssItems[cve]; ok {
+			vulnEPSS = &epss
+		}
+		if len(rhelEPSSDetails) > 0 {
+			// overwrite with RHSA EPSS score if it exists
+			if rhelEPSS, ok := rhelEPSSDetails[name]; ok {
+				vulnEPSS = &rhelEPSS
+			}
+		}
```

Comment:
You are allowed to index into a `nil` map, so we can remove the `len` check

```suggestion
		// overwrite with RHSA EPSS score if it exists
		if rhelEPSS, ok := rhelEPSSDetails[name]; ok {
			vulnEPSS = &rhelEPSS
		}
```